freqtrade/rpc/telegram.py

```diff
@@ -114,8 +114,9 @@ def _init_keyboard(self) -> None:
                                  r'/stopbuy$', r'/reload_config$', r'/show_config$',
                                  r'/logs$', r'/whitelist$', r'/blacklist$', r'/bl_delete$',
                                  r'/weekly$', r'/weekly \d+$', r'/monthly$', r'/monthly \d+$',
-                                 r'/forcebuy$', r'/forcelong$', r'/forceshort$',
-                                 r'/edge$', r'/health$', r'/help$', r'/version$']
+                                 r'/forcebuy$', r'/forcesell$', r'/edge$', r'/health$', r'/help$',
```

Comment:
You'll need to be careful here - this removes `/forcelong` (among others) - which is clearly not the intend (i assume it's a side-effect of merging develop).

Reply:
Thanks for being so patients, will fix and try to be more careful ;-)